tabpy-tools/tabpy_tools/schema.py

```diff
@@ -42,20 +42,23 @@ def _generate_schema_from_example_and_description(input, description):
         # This should not fail unless there are bugs with either genson or
         # jsonschema.
         _validate(input, input_schema)
-    except Exception as e:
+    except Exception:
```

Comment:
Generally want to avoid catching bare Exceptions. Especially here where it's not doing anything of value (a fuller traceback should be logged with appropriate configuration) but again orthogonal to change

Reply:
Opened #212 for this

Reply:
What tool are you using for PEP8 compliance? Ran this with flake8 and no issues there so curious if I should be using something else

Reply:
pep8speak

Reply:
Think we can get away with removing the exception block altogether? If the error propagates will write to stdout with a lot more information (specifically traceback)

Reply:
Took a slightly different approach than what was suggested to use `logger.exception` which would print the stack trace as well.

Ran with pycodestyle 2.5.0 and came back clean. If you still get something on your end can you send me your pycodestyle version so I can try to reproduce?